redux/userSlice.ts

```diff
@@ -1,26 +1,34 @@
 import { createSlice, PayloadAction, Slice } from "@reduxjs/toolkit";
+import { EarnedBadges } from "../graphql/fetchEarnedBadges";
 
 import { Users } from "../graphql/fetchUserProfileCard";
 import { RootState } from "./rootReducer";
 
 export type userState = {
   userList: Users[];
```

Comment:
I wonder if this slice is even needed. I know lucky created it.

Reply:
But I might ask him to remove it

Reply:
I removed it and just used local state.